docs/public/cascara_meta.svg

```diff
@@ -1 +1 @@
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" height="100" width="300" xmlns:v="https://vecta.io/nano"><linearGradient id="A" x1="14.645%" x2="85.355%" y1="85.355%" y2="14.645%"><stop offset="0" stop-color="#90023a"/><stop offset="1" stop-color="#b4104b"/></linearGradient><linearGradient id="B" x1="9.039%" x2="90.961%" y1="78.681%" y2="21.319%"><stop offset="0" stop-color="#a20943"/><stop offset="1" stop-color="#c51653"/></linearGradient><linearGradient id="C" x1="83.742%" x2="17.104%" y1="108.444%" y2="-6.98%"><stop offset="0" stop-color="#00873e"/><stop offset="1" stop-color="#006129"/></linearGradient><linearGradient id="D" x1="50%" x2="50%" y1="100%" y2="0%"><stop offset="0" stop-color="#b4104b"/><stop offset="1" stop-color="#d91c5c"/></linearGradient><g transform="translate(60 30)" fill-rule="evenodd"><path d="M15 12.5a7.5 7.5 0 1 0 15 0 7.5 7.5 0 1 0-15 0" fill="url(#A)"/><path d="M20 25a10 10 0 1 0 20 0 10 10 0 1 0-20 0" fill="url(#B)"/><path d="M2.495 0C1.718.006.942.062.171.171c-.733 5.225.892 10.721 4.914 14.743s9.519 5.649 14.744 4.915c.733-5.225-.892-10.722-4.914-14.744C11.547 1.718 7.147.031 2.737 0z" fill="url(#C)"/><path d="M0 27.5C0 34.403 5.597 40 12.5 40S25 34.403 25 27.5 19.403 15 12.5 15 0 20.597 0 27.5" fill="url(#D)"/><text fill="#333" font-family="AgletSlab-Semibold, Aglet Slab" font-size="42" font-weight="500" letter-spacing="-1.768"><tspan x="44.1" y="34.5">C</tspan><tspan font-size="38" letter-spacing="-1.6" x="68.918" y="34.5">a</tspan><tspan font-size="38" letter-spacing="-1.2" x="88.522" y="34.5">scara</tspan></text></g></svg>
```

Comment:
This SVG file is used in the README.md file. We should not modify it as this will break how the README looks with our logo on dark screens, etc. We should instead make a new, separate SVG file based on the SVG info inside the Logo.js file that was previously being used.

It probably also makes sense to move this file that it used specifically for the README and put it in the repo root with the README. It is not very obvious that the README is using this file that is located within a package.

Reply:
SVG file restored and moved to the root, we also have a new SVG logo in Public which is based on the existing Logo component.